javascript/cwp-pointers.js

```diff
@@ -0,0 +1,20 @@
+(function ($, cwpp){
+
+    $(document).ready( function() {
+
+        $("#cwp_product_affiliate_link").on("keyup", function(e){
+            if($(this).val().indexOf("amazon") != -1){
+                $("#cwp_product_affiliate_link_upsell").show();
```

Comment:
please add a timeout and not show right away, the timeout should be 500ms. be careful to clear it in the bind function, something like this 
```
clearTimeout (timeout);
setTimeout( function (){ show_pointer() },500 );

```